apps/web/package.json

```diff
@@ -42,7 +39,6 @@
     "@types/pg": "^8.11.6",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "dotenv": "^16.4.5",
```

Comment:
Don't remove this one. I need to run CI

Reply:
i did nothing it's duplicated from dependencies and i guess pnpm removed the duplicate